regulations/templates/regulations/layers/table.html

```diff
@@ -2,7 +2,7 @@
     Used to turn markdown tables into HTML tables
 {% endcomment %}
 <div class="table-wrap">
-    <table>
+    <table width="100%">
```

Comment:
Are you sure this is needed? It looks like you've added equivalent CSS in a different PR? If it's needed, please comment why using 
`{% comment %}`